.circleci/config.yml

```diff
@@ -34,11 +31,9 @@ workflows:
   version: 2
   build:
     jobs:
-      - godeps
-      - test-race:
-          # TODO: Remove this alias as part of https://github.com/influxdata/influxdb/issues/22623
```

Comment:
What's the status on that issue?

Reply:
Still open, will need to be done before this can merge. I don't want to submit the request until this is approved since other merges will be blocked when the list of required jobs changes.

Reply:
Oh I see - I was reading that other issue wrong, makes sense.